flocker/node/agents/test/test_blockdevice.py

```diff
@@ -5500,6 +5500,15 @@ def test_current_machine_is_live(self):
             d.addCallback(lambda live:
                           self.assertIn(self.api.compute_instance_id(), live))
             return d
+
+        def test_list_live_nodes(self):
+            """
+            ``list_live_nodes`` returns an iterable of unicode values.
+            """
+            live_nodes = self.api.list_live_nodes()
+            for x in live_nodes:
+                self.expectThat(type(x), Is(unicode))
```

Comment:
:question: No change required, but in case you were curious, testtools also supplies `IsInstance` and `AllMatch` matchers

```
self.assertThat(live_nodes, AllMatch(IsInstance(unicode)))
```